File: JavaFongModel/app/src/main/java/com/example/fong/MainActivity.java
package com.example.fong;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
public class MainActivity extends AppCompatActivity {
    private MySurfaceView mGLSurfaceView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mGLSurfaceView = new MySurfaceView(this);
        setContentView(mGLSurfaceView);
    }
}

File: KotlinNewsApp/app/src/main/java/com/example/kotlinnewsapp/MainActivity.kt
package com.example.kotlinnewsapp

import android.os.Bundle
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.compose.ui.ExperimentalComposeUiApi
import com.example.kotlinnewsapp.ui.theme.KotlinNewsAppTheme

class MainActivity : ComponentActivity() {
    // private val newsViewModel: NewsViewModel by viewModels()

    @OptIn(ExperimentalComposeUiApi::class)
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            KotlinNewsAppTheme() {
                NewsScreen()
            }
        }
    }
}